### DISCUSSION TITLE
Flickering issue when route in nested page

### DISCUSSION THREAD

**Discussion by teamboot88** [nuxt/nuxt#16047] [Questions]
**nuxt version : 3.0.0**

![KakaoTalk_20221208_171834532](https://user-images.githubusercontent.com/54491002/206395193-ea0ffaf5-fb14-4063-a948-275a360d300f.jpg)

When navigating from a nested page to another page, 
the child component is unmounted before parent component.

Therefore, when the child content is larger than the screen size and is scrolled down, 
the content of the child content disappears, so it is **scrolled to the top**. (It is not represented in the above picture)

As a result, the flickering occurs because it is scrolled to the top before navigation.
Is there a solution?

**Comment by Geometrically** [comment]
Did you ever find a solution to this?
